README.md

```diff
@@ -23,6 +23,8 @@ Now you can run the script by typing (You will need netcat installed on the comp
 
     ./init_and_wait_for_db.sh
 
+**You will need to modify config.example.py with the correct database parameters.**
```

Comment:
Wouldn't it be bad for the db container to expose it automatically? Honestly, it should be simple enough to configure the config.example.py with the correct information. I feel like there should be two sets of instructions, installing the db from the script and installing it manually. Just my thoughts.